README.md

```diff
@@ -140,13 +127,17 @@ Set the enviroment variable `OMNECT_RELEASE_IMAGE` to `1` for release builds. Th
 Differences:
 - Release build
   - default firewall config which allows input for established connections only
+  - serial console is disabled
+  - ssh login via public key only

```

Comment:
this is true, if the device is in a local network but if not (and imho that's a more likeley usecase) we use ssh login via bastion host only ... this is different from the classic ssh pubkey